### ISSUE TITLE
The performance of the algorithm is too far from what the paper claims.

### ISSUE THREAD

**Issue by CharlesNord** [ChrisChen1023/HINT#6]
Did you upload the correct pre-trained model? Why is the algorithm performing so poorly? 

![test](https://github.com/ChrisChen1023/HINT/assets/13357806/c9d9b691-9677-4235-9e7f-48e0a8d94803)

**Comment by ChrisChen1023** [comment]
> Did you upload the correct pre-trained model? Why is the algorithm performing so poorly?
> 
> ![test](https://private-user-images.githubusercontent.com/13357806/314919773-c9d9b691-9677-4235-9e7f-48e0a8d94803.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0MjMyMTgsIm5iZiI6MTcxNDQyMjkxOCwicGF0aCI6Ii8xMzM1NzgwNi8zMTQ5MTk3NzMtYzlkOWI2OTEtOTY3Ny00MjM1LTllN2YtNDhlMGE4ZDk0ODAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDIwMzUxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyOWZmMjhhNmVmNDFkOTVlYjY1NGU2N2JmMjY5MTc1NWIzZjY5MmFlNmY1OGUxZmJjMDhhOGRhZTQ4ODVkMWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OrDiDIuTHXt2Q8Q5DVFszNyR2-Kgrto_mzEzt4UOLBw)

Hi there,

Thanks for your interest for our work. We do release the right pre-trained model. We do not use the above images for showcasing the performance in our paper. The artifect could be raised because of the training strategy that we use the irregular masks to cover the whole images for training. We are working on solve that with the next project. Cheers.
